lib/pagy/extras/headers.rb

```diff
@@ -0,0 +1,35 @@
+# See the Pagy documentation: https://ddnexus.github.io/pagy/extras/headers
+# encoding: utf-8
+# frozen_string_literal: true
+
+class Pagy
+  # Add specialized backend methods to add pagination response headers
+  module Backend ; private
+
+    include Helpers
+
+    def pagy_headers_merge(pagy)
+      response.headers.merge!(pagy_headers(pagy))
+    end
+
+    def pagy_headers(pagy)
+      hash = pagy_headers_hash(pagy)
+      { 'Links'    => hash[:links].map{|rel, link| %(<#{link}>; rel="#{rel}")}.join(', ') }.tap do |h|
```

Comment:
more readable/performant to not use `.tap`

Reply:
probably... I will try